src/OpenSearch.Client/Document/Multiple/BulkAll/BulkAllRequest.cs

```diff
@@ -78,7 +78,7 @@ public interface IBulkAllRequest<T> where T : class
 		///<summary>The index to use for items that don't specify one. By default, will be inferred from <typeparamref name="T"/>.
 		/// If no default index has been mapped for <typeparamref name="T" />
 		/// using <see cref="ConnectionSettingsBase{TConnectionSettings}.DefaultMappingFor{TDocument}"/>
-		/// on <see cref="Osc.ConnectionSettings"/>, an exception will be thrown.
+		/// on <see cref="ConnectionSettings"/>, an exception will be thrown.
```

Comment:
cref="Client. ConnectionSettings"

Reply:
That references resolves correctly as is -- both BulkAllRequest and ConnectionSettings are in the same namespace -- OpenSearch.Client.